packages/graphback-core/src/runtime/CRUDService.ts

```diff
@@ -83,7 +83,16 @@ export class CRUDService<T = any> implements GraphbackCRUDService<T>  {
 
   //tslint:disable-next-line: no-any
   public async findBy(filter: any, context: GraphbackContext, orderBy?: GraphbackOrderBy, page?: GraphbackPage): Promise<ResultList<T>> {
-    const items = await this.db.findBy(filter, context, orderBy, page);
+    let count: number;
+    if (context.graphback.options.aggregations?.count) {
+      count = await this.db.count(filter);
+    }
```

Comment:
@wtrocki this is where it will be executed

Reply:
And this will be executed if the gql asks for it. 
e.g 
```
findNotes {
 items {
  id
}
count
}
```
Will execute the count query in db but 
```
findNotes {
 items {
  id
}
}
```

won't

Reply:
Something similar will happen for `items`, if it is not asked in the incoming gql, we won't send the query to db.